addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -917,6 +917,10 @@ var SnippetsMenu = Widget.extend({
             });
 
             var $target = $(srcElement);
+            //  click outside for close dropdown
+            if (!$target.next('we-toggler.active').length && !$target.is('we-toggler.active')) {
```

Comment:
I don't understand this condition. Can you explain? Is not something like `if (!$target.is('we-button, we-toggler')) {` good enough?